CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 7.0.0
+
+* Make it mandatory to provide a period when calling a variable.
+
+When a computation is requested (with *.calculate_output, entities.__call__, *.calculate[_add|_divide], *.compute[_add|_divide]), the argument `period` is no longer optional. Previously, the period of the simulation was used instead of a missing period. It could introduce silently computation errors.
```

Comment:
For consistency with the rest of the changelog, make it a sub-bullet (and maybe split lines before "Previously")